src/main.rs

```diff
@@ -858,13 +907,17 @@ fn main() -> Result<(), Box<dyn Error>> {
         opts.address
     );
 
-    let runtime = tokio::runtime::Builder::new_multi_thread()
+    let rt = tokio::runtime::Builder::new_multi_thread()
         .worker_threads(10)
```

Comment:
Does this mean it can serve 10 prom requests in parallel? Seems a bit much, but hey

Reply:
It means 10 work threads in the thread-pool for all requests of all types (metrics or `/api/v2`).

Reply:
Then maybe that no of worker threads should be configurable via a clap?

Reply:
I actually wonder we don't just default to `#[tokio::main]` on top of main and call it a day in this case?